packages/components/src/dropdown-menu-v2/styles.ts

```diff
@@ -25,12 +25,14 @@ const ITEM_PADDING_BLOCK = space( 2 );
 const ITEM_PADDING_INLINE = space( 3 );
 
 // TODO:
-// - those values are different from saved variables?
-// - should bring this into the config, and make themeable
-// - border color and divider color are different?
-const DEFAULT_BORDER_COLOR = COLORS.gray[ 300 ];
-const DIVIDER_COLOR = COLORS.gray[ 200 ];
-const TOOLBAR_VARIANT_BORDER_COLOR = COLORS.gray[ '900' ];
+// - border color and divider color are different from COLORS.theme variables
```

Comment:
Opened https://github.com/WordPress/gutenberg/issues/64649 to discuss this separately